### PR TITLE
2.6 January RKE2 Release

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

**Review by rancher-max — DISMISSED** [review]
LGTM - versions look correct when compared to https://github.com/rancher/rke2/blob/v1.21.9-rc1%2Brke2r1/Dockerfile#L95-L113 and https://github.com/rancher/rke2/blob/v1.22.6-rc1%2Brke2r1/Dockerfile#L84-L109

File: channels-rke2.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ releases:
   - version: v1.19.16+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.14+rke2r2
+  - version: v1.20.15-rc1+rke2r1
```
Comment (rancher-max) [inline channels-rke2.yaml]: Just update this to be non-rc and then LGTM

**Review by rancher-max — CHANGES_REQUESTED** [review]

File: channels-rke2.yaml (hunk 1 above)
Comment (briandowns) [reply channels-rke2.yaml]: Updated. Thanks!

**Review by rancher-max — APPROVED** [review]
LGTM

**Pull request merged**